src/librustc_typeck/diagnostics.rs

```diff
@@ -124,8 +124,8 @@ be specified exactly one time.
 
 E0063: r##"
 This error indicates that during an attempt to build a struct or struct-like
-enum variant, one of the fields was not provided. Each field should be specified
-exactly once.
+enum variant, one of the fields was not provided. Each field should be
+specified exactly once.
```

Comment:
I didn't even see that. I must have failed my wrapping somewhere... Well, if it is needed, I can undo this. ^^